spec

```diff
@@ -5505,7 +5505,6 @@ beginseg
     name "ovl_En_Invisible_Ruppe"
     compress
     include "build/src/overlays/actors/ovl_En_Invisible_Ruppe/z_en_invisible_ruppe.o"
-    include "build/data/ovl_En_Invisible_Ruppe/ovl_En_Invisible_Ruppe.data.o"
     include "build/data/ovl_En_Invisible_Ruppe/ovl_En_Invisible_Ruppe.reloc.o"
```

Comment:
Since the file is matching completely, the reloc should be updated too
```suggestion
    include "build/src/overlays/actors/ovl_En_Invisible_Ruppe/ovl_En_Invisible_Ruppe_reloc.o"
```